### DISCUSSION TITLE
Ungoogled Chromium has slower performance compared to Chrome on macOS

### DISCUSSION THREAD

**Discussion by styris-ame** [Eloston/ungoogled-chromium#3125] [Q&A]
After installing Ungoogled Chromium 130.0.6723.91 on my M2 MacBook Air using brew, I get the following score on [browserbench](https://browserbench.org/Speedometer3.0):
<img width="1154" alt="Screenshot 2024-12-11 at 7 28 56 PM" src="https://github.com/user-attachments/assets/13d72820-b468-4b81-872f-1282971a92bd" />

However, with Chrome I consistently get a noticeably better score:
<img width="1133" alt="Screenshot 2024-12-11 at 7 29 29 PM" src="https://github.com/user-attachments/assets/7cee7b4c-d08d-4d53-898a-6bf0b5308ff2" />

I'm wondering why this might be considering that they both should be the same performance wise, and both have V8 optimizer enabled with no extensions.

**Comment by PF4Public** [comment]
Perhaps Google builds with PGO, LTO and other optimisations.
